src/components/Search/SearchView.tsx

```diff
@@ -38,7 +54,11 @@ class SearchView extends React.Component {
           {this.searchIcon(18, 18)}
           <input className="search-input"
             value={this.state.searchText}
-            onChange={(event) => this.setState({ searchText: event.target.value })} />
+            onChange={(event) => {
```

Comment:
```suggestion
            onChange={ (event) => {
```
Space before `(event)`

Reply:
formatter removes the space